PostgreSQL.md

```diff
@@ -138,7 +140,23 @@ To activate this feature, do one of the following:
 * set the `WALG_USE_REVERSE_UNPACK`environment variable
 * add the --reverse-unpack flag
 ```
-wal-g backup-fetch ~/extract/to/here LATEST --reverse-unpack
+wal-g backup-fetch /path LATEST --reverse-unpack
+```
+
+#### Redundant archives skipping
+
+With [reverse delta unpack](#reverse-delta-unpack) turned on, you also can turn on redundant archives skipping.
+Since this feature involves both backup creation and restore process, in order to fully enable it you need to do two things:
+
+1. Optional, but recommended. [Enable rating tar ball composer](#rating-composer-mode) for `backup-push`.
```

Comment:
Not recommended, just explain advantages and disadvantages.

Reply:
Changed in https://github.com/wal-g/wal-g/pull/756/commits/9653426e4bd92774d502b99f2b224d6ad3d21954